libs/dao/src/main/java/com/akto/dto/testing/info/CurrentTestsStatus.java

```diff
@@ -0,0 +1,97 @@
+package com.akto.dto.testing.info;
+
+import java.util.List;
+
+
+public class CurrentTestsStatus {
+    
+    public static final String TEST_RUNS_QUEUED = "testRunsQueued";
+    private int testRunsQueued;
+
+    public static final String TOTAL_TESTS_COMPLETED = "totalTestsCompleted";
+    private int totalTestsCompleted;
+
+    public static final String TEST_RUNS_SCHEDULED = "testRunsScheduled";
+    private int testRunsScheduled;
+
+    public static class StatusForIndividualTest {
+
```

Comment:
make this class outside, since keeping it inside makes it usable for this class only. if it is public subclass, then it may be used outside.